src/test/java/api/entities/Login.java

```diff
@@ -0,0 +1,6 @@
+package api.entities;
+
+public class Login {
```

Comment:
The same thing: I would suggest you to delete this class and move the URL to the APIConstants.java class under the 'utils' folder along with other constants.

Reply:
Fixed